k8s/nginx/apptest/tester/tester.sh

```diff
@@ -17,6 +17,13 @@
 set -xeo pipefail
 shopt -s nullglob
 
+export EXTERNAL_IP="$(kubectl get service/${APP_INSTANCE_NAME}-nginx-svc \
+  --namespace ${NAMESPACE} \
+  --output jsonpath='{.status.loadBalancer.ingress[0].ip}')"
+
 for test in /tests/*; do
-  testrunner -logtostderr "--test_spec=${test}"
+  testspec="$(mktemp XXXXXXXX.yaml)"
```

Comment:
Please rename `XXXXXXXX.yaml` to some more descriptive - `parsed/tmp/temporary.yaml` or so on.

Reply:
`mktemp` will replace `XXXXXXXX` with a random string. I would like to have random string instead of static file name.